.github/workflows/1x-integration-test.yaml

```diff
@@ -1,8 +1,37 @@
-name: Test
+name: 1.x Integration Tests
 
-on: [push]
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'next'
+  push:
+    branches:
+      - '*'
+    tags:
+      - '1.*.*'
 
 jobs:
+  # Golang Integration tests, using the legacy manager/1.x CLI
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16'
+    - name: cache go modules
+      uses: actions/cache@v2.1.6
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go
```

Comment:
This won't work because the restore key is not a prefix of the storing key. It has apparently been broken before this PR, so creating a separate issue might be an OK option here.

Reply:
https://github.com/Kong/kubernetes-ingress-controller/issues/1414